src/plugins/MediaManager.js

```diff
@@ -156,28 +160,32 @@ module.exports = class MediaManager {
 
     /**
      * Stop client webcam video stream.
+     * @returns {boolean} true on success, false on fail
      */
     stopVideoStreaming() {
         if (!this.videoStreaming) {
-            return;
+            return false;
         }
         log.debug('removed local video stream');
-        this.removeVideoStream(this.localVideoStream);
+        const result = this.removeVideoStream(this.localVideoStream);
```

Comment:
Same thing here, This call renegotiateWebRTCConnection which call this.peerConnection.createOffer.